packages/paywall/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@unlock-protocol/paywall",
-  "version": "0.1.4",
+  "version": "0.2.4",
```

```suggestion
  "version": "0.2.0",
```

If we're bumping, let's reset the patch too :)